packages/core/src/components/Dialog/Dialog.jsx

```diff
@@ -102,12 +136,17 @@ Dialog.propTypes = {
    * it (e.g. `document.getElementById(..)`)
    */
   getApplicationNode: PropTypes.func,
+  /**
+   * Additional classes to be added to the header element.
```

Comment:
I wonder if "header element" is confusing here, since we don't mention "header" anywhere else for this component. Should we clarify this by saying...

> Additional classes to be added to the header, which wraps the title and close button.